tests/test_tokenization_common.py

```diff
@@ -3812,7 +3812,9 @@ def test_push_to_hub_dynamic_tokenizer(self):
 
             with open(os.path.join(tmp_dir, "tokenizer_config.json")) as f:
                 tokenizer_config = json.load(f)
-            self.assertEqual(tokenizer_config["auto_map"], ["custom_tokenization.CustomTokenizer", None])
+            self.assertDictEqual(
+                tokenizer_config["auto_map"], {"AutoTokenizer": ["custom_tokenization.CustomTokenizer", None]}
```

Comment:
This illustrates the change: auto_map becomes a dict with several classes (to be able to include the AutoProcessor).